subsys/bluetooth/common/Kconfig

```diff
@@ -118,7 +118,44 @@ config BT_DEBUG_MONITOR
 	  UART_CONSOLE needs to be disabled (in which case printk/printf
 	  will get encoded into the monitor protocol).
 
-endchoice
+config BT_DEBUG_MONITOR_RTT
+	bool "Monitor protocol over RTT"
+	depends on USE_SEGGER_RTT
+	depends on SEGGER_RTT_MAX_NUM_UP_BUFFERS >= 2
+	select BT_DEBUG
+	help
+	  Use a custom logging protocol over the RTT buffer instead of
+	  plain-text output. Requires a special application
+	  on the host side that can decode this protocol. Currently
+	  the 'btmon' tool from BlueZ is capable of doing this.
+
+if BT_DEBUG_MONITOR_RTT
+
+config BT_DEBUG_MONITOR_RTT_BUFFER
+	int "Buffer number used for custom logger output."
+	range 1 SEGGER_RTT_MAX_NUM_UP_BUFFERS
```

Comment:
I read that buffer 0 is configured on compile-time and reserved for Terminal, so I limited the range to start at 1.
Is there any way to end the range at SEGGER_RTT_MAX_NUM_UP_BUFFERS-1?
Syntax: range <symbol> <symbol>
does not allow this directly.